quickstarts/apache-traffic-server/config.yml

```diff
@@ -0,0 +1,52 @@
+  By leveraging New Relic, you can gain a more in-depth understanding of Monitoring Apache Traffic Server is an essential procedure that provides priceless information about the functionality, performance, and general health of this potent web caching and proxying solution. You may obtain a thorough insight of ATS's behaviour, spot any problems before they become serious, and make sure that your web infrastructure is working as efficiently as possible by closely monitoring it. 
+
+  It's a proactive strategy for guaranteeing outstanding web performance, exceptional user experiences, and the dependable functioning of your web infrastructure. You can efficiently manage ATS and keep up a solid, high-performance web infrastructure by New Relic ATS.. 
+
+
+  ## What’s included in this quickstart?
+  The New Relic Apache Traffic Server monitoring quickstart provides out-of-the-box reporting that covers the following areas:
+
+  - Real-time visibility: Get insights into the performance of your ATS deployment in real time.
+  - Optimized performance: Identify performance bottlenecks and optimize your ATS configuration to improve performance.
+  - Early issue detection: Get alerts when there are potential problems with your ATS deployment so you can take action quickly.
+  - Capacity planning: Plan for future growth by understanding how your ATS deployment is currently being used.
+  - Efficient troubleshooting: Troubleshoot problems with your ATS deployment quickly and easily by using New Relic's tools.
+  - User experience enhancement: Improve the user experience of your applications by understanding how they are being used and making changes to improve performance.
+  - Security insights: Get insights into the security of your ATS deployment and identify potential threats.
+  - Continuous improvement: Use New Relic's data to continuously improve the performance and security of your ATS deployment.
+  - Business impact: Understand the impact of your ATS deployment on your business by measuring metrics such as revenue, traffic, and customer satisfaction.
+
+  A well-monitored Apache Traffic Server (ATS) can help you improve the performance of your online services, which can lead to increased user engagement, longer site visits, and higher profits.
+  
+summary: |
+  Optimize your Apache Traffic Server performance using New Relic Apache Traffic Server monitoring.
+icon: logo.png
+level: New Relic
+authors:
+  - New Relic
+title: Apache Traffic Server
+documentation:
+  - name: Documentation on how to integrate Apache Traffic Server
+    description: |
+      Monitor the performance metrics of your Apache Traffic Server instances in real-time with New Relic Apache Traffic Server.
+    url: >-
+      https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/apache-traffic-server-monitoring-integration/
+keywords:
+  - apache-traffic-server
+  - apache
+  - ats
+  - NR1_addData
```

Comment:
```suggestion
```

here as well

Reply:
Hi @mdumpati,
As per your suggestions, I have removed the keyword. Kindly review it.

Thanks.